spec/observables/from-promise-spec.js

```diff
@@ -32,4 +108,25 @@ describe('Observable.fromPromise', function () {
       done();
     });
   });
+
+  it('should globally throw unhandled errors', function (done) {
+    var invoked = false;
+    process.on('uncaughtException', function (reason, p) {
+      if (invoked) {
+        return;
+      }
+      invoked = true;
+      expect(reason).toBe('fail');
+      done();
+    });
+
+    Observable.fromPromise(Promise.reject('bad'))
+      .subscribe(
+        done.fail,
+        function (e) {
+          expect(e).toBe('bad');
+          throw 'fail';
```

Comment:
Seems this test is for `onError` behavior of `subscribe`, does PromiseObservable has special handling for this case?